src/commands/text.ts

```diff
@@ -31,6 +31,11 @@ export const textCommands: CommandEntry[] = [
     },
     {
         name: 'mind-reader.getWordsUnderCursor',
+        callback: runCursorContext,
+    },
+    {
+        name: 'mind-reader.toggleTextToSpeech',
+        callback: toggleTTS,
         callback: runCursorContext
```

Comment:
probably need to remove this. can't have two callbacks, ide was angry about that